lib/jsdom/events/EventTarget.js

```diff
@@ -0,0 +1,189 @@
+﻿"use strict";
+
+var EventException = require("./EventException");
+
+var propEvents = Symbol();
+
+module.exports = EventTarget;
+
+function EventTarget() {
+  if (!this.__jsdom) {
```

Comment:
No, this is only necessary for windows, but Window inherits from EventTarget. How'd you special case Window?